iis/datadog_checks/iis/iis.py

```diff
@@ -3,8 +3,8 @@
 # Licensed under Simplified BSD License (see LICENSE)
 from six import iteritems
 
-from datadog_checks.base import PDHBaseCheck, is_affirmative
-from datadog_checks.base.utils.containers import hash_mutable
+from datadog_checks.base import PDHBaseCheck
+from datadog_checks.base.stubs import datadog_agent
```

Comment:
That doesn't look right? You don't want to use the stub version.

Reply:
I agree. I copied it from [vsphere](https://github.com/DataDog/integrations-core/blob/master/vsphere/datadog_checks/vsphere/vsphere.py#L17) which seems to be the only other case outside of datadog_checks_base

Should it be 
```
try:
    import datadog_agent
except ImportError:
    from .stubs import datadog_agent
```
? cc @FlorianVeaux